src/Our.Umbraco.Ditto/Attributes/UmbracoPropertyAttribute.cs

```diff
@@ -51,5 +52,10 @@ public UmbracoPropertyAttribute(string propertyName, string altPropertyName = ""
         /// Gets or sets the default value.
         /// </summary>
         public object DefaultValue { get; set; }
+
+        /// <summary>
+        /// Defines if property is an Archetype
+        /// </summary>
+        public bool IsArchetype { get; set; }
```

Comment:
@leekelleher Type converters makes sense to me, could potentially extend the ArchetypeModel (https://github.com/imulus/Archetype/blob/master/app/Umbraco/Umbraco.Archetype/Models/ArchetypeModel.cs) with a FieldSetAs<T> method of something? Would these TypeConverters be completed in a separate assembly to avoid assembly dependency?